lib/internal/url.js

```diff
@@ -191,7 +209,7 @@ class URLSearchParams {
         this[searchParams] = childParams.slice();
       } else if (method !== null && method !== undefined) {
         if (typeof method !== 'function') {
-          throw new ERR_ARG_NOT_ITERABLE('Query pairs');
+          throwTypeError(new ERR_ARG_NOT_ITERABLE('Query pairs'));
```

Comment:
If you look at `lib/internals/errors.js`, you'll see that `ERR_ARG_NOT_ITERABLE` is already a `TypeError`.